samples/README.md

```diff
@@ -1,325 +1,195 @@
 [//]: # "This README.md file is auto-generated, all changes to this file will be lost."
-[//]: # "To regenerate it, use `npm run generate-scaffolding`."
+[//]: # "To regenerate it, use `python -m synthtool`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# Google Cloud Storage: Node.js Samples
+# [Google Cloud Storage: Node.js Samples](https://github.com/googleapis/nodejs-storage)
 
 [![Open in Cloud Shell][shell_img]][shell_link]
 
-[Cloud Storage](https://cloud.google.com/storage/docs) allows world-wide storage and retrieval of any amount of data at any time. You can use Google Cloud Storage for a range of scenarios including serving website content, storing data for archival and disaster recovery, or distributing large data objects to users via direct download.
+> Node.js idiomatic client for [Cloud Storage][product-docs].
+
+[Cloud Storage](https://cloud.google.com/storage/docs) allows world-wide
+storage and retrieval of any amount of data at any time. You can use Google
+Cloud Storage for a range of scenarios including serving website content,
+storing data for archival and disaster recovery, or distributing large data
+objects to users via direct download.
 
 ## Table of Contents
 
 * [Before you begin](#before-you-begin)
 * [Samples](#samples)
-  * [ACL (Access Control Lists)](#acl-access-control-lists)
+  * [Acl](#acl)
+  * [Bucket Lock](#bucket-lock)
   * [Buckets](#buckets)
   * [Encryption](#encryption)
   * [Files](#files)
+  * [Iam](#iam)
   * [Notifications](#notifications)
+  * [Quickstart](#quickstart)
   * [Requester Pays](#requester-pays)
-  * [Bucket Lock](#bucket-lock)
 
 ## Before you begin
 
 Before running the samples, make sure you've followed the steps in the
-[Before you begin section](../README.md#before-you-begin) of the client
+[Using the client library](https://github.com/googleapis/nodejs-storage#using-the-client-library) of the client
```

Comment:
This doesn't read very well. Maybe it should be quoted around the link, and we can probably remove all the "client library" phrasing. Maybe "...make sure you've followed [these steps](#link)."